src/test/resources/projects/basic/archetype.properties

```diff
@@ -3,8 +3,8 @@ applicationName=ShopApplication
 groupId=hu.blackbelt
 artifactId=shop
 package=hu.blackbelt.shop
-jslSpringbootVersion=JSL-SPRINGBOOT-VERSION-PLACEHOLDER
-jslTatamiVersion=JSL-TATAMI-VERSION-PLACEHOLDER
+jslSpringbootVersion=1.0.4.20230221_041641_66ea6c71_develop
+jslTatamiVersion=1.1.4.20230221_041625_ba9cc1a5_develop
```

Comment:
This looks a bit suspicious part 2

Reply:
It is replkaced by compilayion. Do not commit these changes.

Reply:
[9b0620e](https://github.com/BlackBeltTechnology/judo-jsl-springboot-archetype/pull/31/commits/9b0620e949946d45b3567e895c56a555455b4774)